kolibri/plugins/user/management/commands/background.py

```diff
@@ -56,23 +39,27 @@ def handle(self, *args, **options):
                 )
             )
 
-            new_img_path = os.path.abspath(os.path.expanduser(options["destination"]))
-            if not os.path.exists(new_img_path):
+            new_img = os.path.abspath(os.path.expanduser(options["new-image"]))
```

Comment:
I know this is existing functionality, but looks like we don't validate the file extension/mimetype?

Reply:
this should work with png, svg, jpg, gif, etc... I had intentionally left it permissive